.github/workflows/scan.yml

```diff
@@ -8,24 +8,33 @@ on:
       - "rc"
       - "hotfix-rc"
   pull_request:
```

Comment:
I don't believe we need pull_request, target should run on both forks and local PRs just in a different context https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

Reply:
Based on my testing it didn't work, but I'll remove that for the moment and we merge to see what really happens.